call.go

```diff
@@ -122,6 +122,9 @@ func sendRequest(ctx context.Context, dopts dialOptions, compressor Compressor,
 	if err != nil {
 		return nil, Errorf(codes.Internal, "grpc: %v", err)
 	}
+	if len(outBuf) > msgSizeLimit {
+		return nil, Errorf(codes.InvalidArgument, "Sent message larger than max (%d vs. %d)", len(outBuf), msgSizeLimit)
```

Comment:
gRPC itself should not return code `InvaludArgument`.
(If this is consistent with other languages, probably we should comment that).